content/zotero_intro_to.md

```diff
@@ -292,7 +292,7 @@ Example : the ISBN of a book  or a report, the DOI of a scientific paper.
 
 If you already have stored PDFs on your computer, it might be possible to retrieve metadata from them afterwards.
 
-**/!\\ This is not the recommended default method**.  It is useful for processing a bunch of PDFs you have in stock, but this should not be your usual method of adding content to your Zotero library.
+**⚠️ This is not the recommended default method**.  It is useful for processing a bunch of PDFs you have in stock, but this should not be your usual method of adding content to your Zotero library.
```

Comment:
to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

Reply:
Ok